### PR TITLE
Add Graylog icon and Windows uninstaller entry for 64 bit

### PR DESCRIPTION
**Pull request by nicozanf**
it needs the graylog.ico file in the /dist folder

### CODE REVIEW

File: dist/recipe.nsi [hunk 1]
```diff
@@ -47,7 +47,7 @@
 ;--------------------------------
 ;Folder selection page
  
-  InstallDir "$PROGRAMFILES64\graylog\collector-sidecar"
+  InstallDir "$PROGRAMFILES64\Graylog\collector-sidecar"
```
Comment (mariussturm) [inline dist/recipe.nsi]: Changing the installation path needs to be handled in case a user is updating the software

File: dist/recipe.nsi [hunk 2]
```diff
@@ -77,14 +78,40 @@
 Section "Install"
  
   ;Add files
-  SetOutPath "$INSTDIR\generated"  
   SetOutPath "$INSTDIR"
```
Comment (mariussturm) [inline dist/recipe.nsi]: If we remove the `\generated` we have to make sure that is will be created at some other point.

Comment (mariussturm) [inline dist/recipe.nsi]: Yes, the Sidecar is dropping the generated configurations there. After the first start you can find some files there.

Comment (nicozanf) [inline dist/recipe.nsi]: ok, I'm changing it back

**Pull request merged**